homeassistant/components/vicare/types.py

```diff
@@ -64,6 +64,55 @@ def from_ha_preset(
 }
 
 
+class VentilationMode(enum.StrEnum):
```

Comment:
Things that are only used in a single module are better kept in that module.

Reply:
moved in https://github.com/home-assistant/core/pull/124613